ml/src/main/scala/org/apache/spark/ml/h2o/algos/H2OAutoML.scala

```diff
@@ -107,9 +107,12 @@ class H2OAutoML(override val uid: String) extends Estimator[H2OMOJOModel]
         " your 'excludeAlgo', 'maxModels' or 'maxRuntimeSecs' properties.") with NoStackTrace
     }
 
-    val mojoData = ModelSerializationSupport.getMojoData(aml.leader())
+    val binaryModel = aml.leader()
+    val mojoData = ModelSerializationSupport.getMojoData(binaryModel)
     val model = H2OMOJOModel.createFromMojo(mojoData, Identifiable.randomUID(aml.leader()._parms.algoName()))
-    model.setConvertUnknownCategoricalLevelsToNa(true)
```

Comment:
I set default to true on automl and grieadseach so we have one place where this value is defined